Extensions/TiledSpriteObject/tiledspriteruntimeobject.js

```diff
@@ -43,6 +43,20 @@ gdjs.TiledSpriteRuntimeObject = function(runtimeScene, tiledSpriteObjectData)
 gdjs.TiledSpriteRuntimeObject.prototype = Object.create( gdjs.RuntimeObject.prototype );
 gdjs.registerObject("TiledSpriteObject::TiledSprite", gdjs.TiledSpriteRuntimeObject);
 
+gdjs.TiledSpriteRuntimeObject.prototype.updateFromObjectData = function(oldObjectData, newObjectData) {
```

Comment:
@Bouh All objects/behaviors got these new methods to do the "diff" between the old and new data at their level (ideally in the future we could scrap these methods from exported game to save a bit of size).